config/peerpods/local/configuration-remote.toml

```diff
@@ -13,7 +13,8 @@
 
 [hypervisor.remote]
 remote_hypervisor_socket = "/run/peerpod/hypervisor.sock"
-remote_hypervisor_timeout = 600
+remote_hypervisor_timeout = 900
+disable_guest_selinux = true
```

Comment:
why is this needed?

Reply:
This is there in the current config. In tests we have seen sometimes the timeout required is higher. Also selinux is disabled in the guest for now to avoid issues with accessing the TPM device. More work is needed to enable selinux for the Pod VM